src/components/ModelingSidebar/ModelingPanes/KclEditorPane.tsx

```diff
@@ -210,12 +210,17 @@ export const KclEditorPane = () => {
             editorManager.setEditorView(_editorView)
             kclEditorActor.send({ type: 'setKclEditorMounted', data: true })
 
-            // On first load of this component, ensure we show the current errors
-            // in the editor.
-            // Make sure we don't add them twice.
-            if (diagnosticCount(_editorView.state) === 0) {
-              kclManager.setDiagnosticsForCurrentErrors()
-            }
+            // Update diagnostics as they are cleared when the editor is unmounted.
+            // Without this, errors would not be shown when closing and reopening the editor.
+            // eslint-disable-next-line @typescript-eslint/no-floating-promises
```

Comment:
Assuming this is actually safe, instead of ignoring the lint, use `void` ... `.catch(reportRejection)`.

I think that in an ideal world, we'd be more careful about canceling or ignoring the result of async work. E.g. what if they close the editor after calling `safeParse()` but before it resolves? We'd still run the handler. But this is probably fine.